apps/tangle-dapp/app/liquid-staking/[tokenSymbol]/page.tsx

```diff
@@ -1,28 +1,53 @@
+'use client';
+
 import { notFound } from 'next/navigation';
-import { FC } from 'react';
+import { FC, useState } from 'react';
 
-import LiquidStakeAndUnstakeCards from '../../../components/LiquidStaking/LiquidStakeAndUnstakeCards';
 import { LiquidStakingSelectionTable } from '../../../components/LiquidStaking/LiquidStakingSelectionTable';
-import UnstakeRequestsTable from '../../../components/LiquidStaking/UnstakeRequestsTable';
+import LiquidStakeCard from '../../../components/LiquidStaking/stakeAndUnstake/LiquidStakeCard';
+import LiquidUnstakeCard from '../../../components/LiquidStaking/stakeAndUnstake/LiquidUnstakeCard';
+import UnstakeRequestsTable from '../../../components/LiquidStaking/unstakeRequestsTable/UnstakeRequestsTable';
 import isLiquidStakingToken from '../../../utils/liquidStaking/isLiquidStakingToken';
+import TabListItem from '../../restake/TabListItem';
+import TabsList from '../../restake/TabsList';
 
 type Props = {
   params: { tokenSymbol: string };
 };
 
 const LiquidStakingTokenPage: FC<Props> = ({ params: { tokenSymbol } }) => {
+  const [isStaking, setIsStaking] = useState(true);
+
   if (!isLiquidStakingToken(tokenSymbol)) {
     return notFound();
   }
 
   return (
     <div className="grid grid-cols-2 gap-12">
-      <LiquidStakeAndUnstakeCards />
+      <div className="flex flex-col gap-4 w-full min-w-[450px] max-w-[600px]">
+        <TabsList className="w-full">
+          <TabListItem
+            href="#"
```

Comment:
Hey there! Just a heads-up, I think it would be great to make the `href` prop optional. If `href` is provided, we can render the `Link` component; otherwise, we'll render a standard button component. No rush, we can handle it in another PR.